src/com/tw/go/plugin/nuget/NuGetFeedDocument.java

```diff
@@ -69,7 +75,7 @@ public PackageRevision getPackageRevision(boolean lastVersionKnown) {
 
     private String getReleaseNotes() {
         String releaseNotes = getProperty(getProperties(), "ReleaseNotes");
-        if (releaseNotes == null || releaseNotes.trim().isEmpty()) return null;
```

Comment:
Sorry for the confusion. You just need to add
```java
assertThat(result.getTrackbackUrl(), is(""));
```
since you already have
```java
assertThat(result.getRevisionComment(), is(""));
```